it/mail/chrome/messenger/messenger.properties

```diff
@@ -791,45 +791,45 @@ trialUpgradeButton.label=Acquista Postbox...
 # invalidValidateInfo1.padding is how many pixels to move down invalidValidateInfo2
 invalidValidateInfo2.padding=15
-invalidValidateInfo2.label=Per acquistare un codice di licenza visita il nostro negozio online cliccando sul pulsante Purchase Postbox qui sotto.
-
-timebombExpiringDaysNotification.title=This out of date version of Postbox is going to expire in %1$S days.
-timebombExpiringDayNotification.title=This out of date version of Postbox is going to expire in %1$S day.
-timebombExpiringTodayNotification.title=This out of date version of Postbox is going to expire today.
-timebombExpiredNotification.title=This out of date version of Postbox has expired.
-timebombExpiredNotificationFree.title=Need More Accounts?
-timebombExpiringNotificationUpgrade.label=Download Postbox...
+invalidValidateInfo2.label=Per acquistare un codice di licenza visita il nostro negozio online cliccando sul pulsante Acquista Postbox qui sotto.
+
+timebombExpiringDaysNotification.title=Questa versione non aggiornata di Postbox scadrà tra %1$S giorni.
+timebombExpiringDayNotification.title=Questa versione non aggiornata di Postbox scadrà tra %1$S giorni.
+timebombExpiringTodayNotification.title=Questa versione non aggiornata di Postbox scadrà oggi.
+timebombExpiredNotification.title=Questa versione non aggiornata di Postbox è scaduta.
+timebombExpiredNotificationFree.title=Hai bisogno di più account?
+timebombExpiringNotificationUpgrade.label=Scarica Postbox...
 timebombExpiringNotificationUpgrade.accessKey=D
-timebombWindow.title=Out of Date Version Expired
-timebombUpgradeButton.label=Download Postbox...
-timebombUpgradeButtonFree.label=Upgrade
+timebombWindow.title=Versione non aggiornata scaduta
+timebombUpgradeButton.label=Scarica Postbox...
+timebombUpgradeButtonFree.label=Aggiorna
 timebombContinueButton.label=Non in linea
-timebombContinueButtonFree.label=No, Thanks
-timebombEditButton.label=Edit License
-timebombValidateInfo1.label=You can still run Postbox offline and view or export your data, but you will not be able to run Postbox online until you download a new version.  Please update Postbox to the latest version by clicking the "Download" button below.
-timebombValidateInfo1Free.label=Upgrade to Postbox to get unlimited account access.  Setup as many POP or IMAP accounts as you like.  Then, view them together within Postbox's Unified Views.
+timebombContinueButtonFree.label=No, grazie
+timebombEditButton.label=Modifica licenza
+timebombValidateInfo1.label=Puoi ancora eseguire Postbox offline e visualizzare o esportare i tuoi dati, ma non potrai eseguire Postbox online fino a che non scarichi una nuova versione.  Si prega di aggiornare Postbox alla versione più recente cliccando il pulsante "Download" sottostante.
+timebombValidateInfo1Free.label=Effettua l'upgrade a Postbox per avere accesso a account illimitati. Cnfigura quanti account POP o IMAP desideri. Poi, visualizzali insieme all'interno delle Visualizzazioni unificate di Postbox.
```

Comment:
CORRECTIONS NEEDED. Please replace the original sentence with the following one: 

+timebombValidateInfo1Free.label=Effettua l'upgrade a Postbox per avere accesso ad account illimitati. Configura tutti gli account POP o IMAP che desideri, poi visualizzali insieme all'interno delle Visualizzazioni unificate di Postbox.